django/thunderstore/repository/models/package.py

```diff
@@ -40,7 +40,7 @@ def get_package_dependants_list(package_pk: int):
 class Package(models.Model):
     objects = PackageQueryset.as_manager()
     owner = models.ForeignKey(
-        "repository.UploaderIdentity",
+        "repository.Namespace",
```

Comment:
Package's owner should still be `team`, and there should be a new field added called `namespace` pointing to the correct `Namespace` instance.